### DISCUSSION TITLE
Call jump event on blueprints causes a crash

### DISCUSSION THREAD

**Discussion by davgmz** [Sixze/ALS-Refactored#125] [Q&A] [answered]
Is there a different jump event than the normal default jump event? I made my character inherit directly from AlsCharacter and transferred the logic from AlsExample character inputs to blueprints, but every time I try to jump the editor crashes. 
List of errors and lines in the crash report,
Unhandled Exception: EXCEPTION_ACCESS_VIOLATION writing address 0x0000000000000674
UnrealEditor_ALS!AAlsCharacter::OnJumpedNetworked() [C:\Users\dav\Documents\Unreal Projects\Als\Plugins\ALS\Source\ALS\Private\AlsCharacter.cpp:1200]
UnrealEditor_ALS!AAlsCharacter::OnJumped_Implementation() [C:\Users\dav\Documents\Unreal Projects\Als\Plugins\ALS\Source\ALS\Private\AlsCharacter.cpp:1183]
UnrealEditor_ALS!UAlsCharacterMovementComponent::ControlledCharacterMove() [C:\Users\dav\Documents\Unreal Projects\Als\Plugins\ALS\Source\ALS\Private\AlsCharacterMovementComponent.cpp:220]

**Comment by Sixze** [comment, marked as answer]
It looks like you didn't assign a valid animation blueprint in the skeletal mesh component.

**Reply by davgmz** [reply]
It has a valid animation plane, it's just a normal AnimInstance. Since I'm just testing, I haven't yet made my animation plane inherit from AlsAnimInstance. That's the problem?

**Reply by Sixze** [reply]
Yes, your animation blueprint should inherit from `UAlsAnimationInstance`.

**Reply by davgmz** [reply]
Thanks, I'll try it later.
